packages/edit-site/src/components/sidebar-navigation-screen/style.scss

```diff
@@ -18,7 +18,7 @@
 .edit-site-sidebar-navigation-screen__content {
 	padding: 0 $grid-unit-20;
 
-	.components-item-group {
+	.edit-site-sidebar-navigation-screen-main {
```

Comment:
Can we create a new `packages/edit-site/src/components/sidebar-navigation-screen-main/style.scss` file and move this style into it?